clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -268,32 +281,31 @@ void testPollResultTimer() {
 
     @Test
     void testMaximumTimeToWait() {
+        List<Optional<? extends RequestManager>> list = new ArrayList<>();
+        list.add(Optional.of(heartbeatRequestManager));
         // Initial value before runOnce has been called
         assertEquals(ConsumerNetworkThread.MAX_POLL_TIMEOUT_MS, consumerNetworkThread.maximumTimeToWait());
+
+        when(requestManagers.entries()).thenReturn(list);
```

Comment:
Minor, but it seems like this line could be more succinct like:

```suggestion
        when(requestManagers.entries()).thenReturn(Collections.singletonList(Optional.of(heartbeatRequestManager)));
```